packages/f/fxdiv/xmake.lua

```diff
@@ -8,12 +7,10 @@ package("fxdiv")
     add_urls("https://github.com/Maratyszcza/FXdiv.git")
     add_versions("2020.12.09", "63058eff77e11aa15bf531df5dd34395ec3017c8")
 
-    add_deps("cmake")
-    on_install("windows", "macosx", "linux", function (package)
-        local configs = {"-DFXDIV_BUILD_TESTS=OFF", "-DFXDIV_BUILD_BENCHMARKS=OFF"}
-        import("package.tools.cmake").install(package, configs)
+    on_install("windows", "macosx", "linux", function(package)
+        os.cp("include", package:installdir())
```

Comment:
Please use the cmake that comes with the library to install it if possible, even if it is header only. This is because cmake also installs an additional xxx.cmake import file so that other cmake projects that depend on it can find it automatically.